test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -37,8 +41,17 @@ describe.skip('Create token, approve token and approve token without gas', funct
           await driver.waitForSelector({ text: 'Create Token', tag: 'button' });
```

Comment:
I think we can remove this line, as the below `clickElement` method will make sure to `findClickableElement` first

Reply:
Done